yt_dlp/extractor/ntvru.py

```diff
@@ -131,12 +152,28 @@ def _real_extract(self, url):
             formats.extend(self._extract_mpd_formats(
                 dash_manifest, video_id, mpd_id='dash', fatal=False))
 
+        metadata = self._download_xml(
+            f'https://www.ntv.ru/exp/video/{video_id}', video_id, 'Downloading XML metadata', fatal=False)
+
         return {
-            'id': xpath_text(video, './id'),
-            'title': title,
-            'description': strip_or_none(unescapeHTML(xpath_text(player, './data/description'))),
-            'thumbnail': xpath_text(video, './splash'),
-            'duration': int_or_none(xpath_text(video, './totaltime')),
-            'view_count': int_or_none(xpath_text(video, './views')),
+            'id': video_id,
             'formats': formats,
+            **traverse_obj(metadata, {
+                'title': ('{*}title/text()', ..., {unescapeHTML}, any),
+                'description': ('{*}description/text()', ..., {unescapeHTML}, any),
```

Comment:
just a note about `unescapeHTML` (that maybe you're already aware of):

it has a [nasty assert](https://github.com/yt-dlp/yt-dlp/blob/5767fb4ab108dddb07fc839a3b0f4d323a7c4bea/yt_dlp/utils/_utils.py#L814) and should always be preceded by `{str}` in a traversal path (or you should just use `{clean_html}` instead)

here it doesn't really matter cuz we're in XML and using `text()`, but this gave me a flashback and I wanted to pass along the pain